### DISCUSSION TITLE
How to add blog to an existing site/wiki?

### DISCUSSION THREAD

**Discussion by ghost** [datenstrom/community#318] [Ask a question] [answered]
Hi, I recently discover Yellow and I'm using it for a new version of my website.

I need static pages + blog pages but I don't know how to have both on same install. I've installed with the site config.

Thanks fo help!

**Comment by schulle4u** [comment]
Could you please explain a little more detailed how your setup looks like? For a static html website you can use [the appropriate build command](https://developers.datenstrom.se/help/server-configuration#static-website). Weblog pages afaik are also included in this process. Using for example a hourly chron job you can have a static website which is regularly updated with your latest blog content. 

If you would like to run Yellow as php website engine and just need static pages in terms of pages with content that isn't updated regularly, please refer to [the detailed help page](https://developers.datenstrom.se/help/how-to-make-a-website) for further information how to make a website, blog and wiki. 

HTH
Steffen

**Comment by ghost** [comment]
Hi!
I will try to explain more clearly (with my poor English !)... It's not a problem of hosting because I use an external hoster. Yellow is working properly.

But at install, I've choose "Site". Logically, the script has create a simple website, and a wiki. (The nav: home, wiki, about).
But finally, I want also create a blog in the same install. But, I've no template, no entry for blog. How to remedy?

Thanks!

**Comment by schulle4u** [comment, marked as answer]
Ah, that's quite easy. Just install the [blog plugin](https://github.com/datenstrom/yellow-plugins/tree/master/blog) from the plugins repository, drop the `blog.zip` into your `system/plugins` folder and open your website in a browser. The blog plugin automatically will install everything you need without damaging the rest of the site. :)

Steffen

**Comment by ghost** [comment]
Thanks, it works perfectly! :)
